src/better-help/index.ts

```diff
@@ -5,6 +5,8 @@ import { Searcher } from './search'
 export function InitHelp(robot: Robot, scripts: IScriptsMap, searcher: Searcher) {
   // -------------- RESPOND WITH HELP FROM HUBOT SCRIPTS ---------------------- //
 
+  const robotName = robot.alias || (robot.name + ' ')
```

Comment:
In fact, hubot-help seems to do something much more specific here:

https://github.com/hubotio/hubot-help/blob/bb3abbc3f380fe3dc7d20573af71d540b30ffd36/src/help.js#L110-L116

In particular, only 1-character aliases have their post-space removed. Here, we assume the alias won't have a space after it, sort of. Wondering if we align this behavior completely?

Reply:
ohhhhhh. Yes. Good call, thank you.

Reply:
Wow, realizing that I _way_ dropped the ball on this, a long time ago. Updating now.

Reply:
Addressed in [423f58cfe6e67d](https://github.com/thesis/hubot-even-better-help/pull/4/commits/423f58cfe6e67dde86ea3d3a3a71c6fce6ee77cc)

Reply:
While updating this, I noticed that the descriptions were pulling in the alias too, since we set the `robotName` const to prefer the alias over the name, if present. But in the context of the descriptions, the name makes more sense. Updated [here](https://github.com/thesis/hubot-even-better-help/pull/4/commits/39e574712f0669bc4e8c4d9386f4f24100b5ae85), and tested in [this thread](https://www.flowdock.com/app/cardforcoin/playground/threads/aBlffWXWH41bdN26D8CwNNUdRuE)